notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1224 @@
+{
```

Comment:
<p>Could you add a short description here about the custom component here itself? Something like: </p><p>We'll build a component that expects a list of 'generators' at initialization, and we will add a new <code>_arun</code> function that... etc. </p><p> </p><p>The topic can be quite new to people we can do some handholding? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
done